package.json

```
@@ -22,17 +22,17 @@
   },
   "scripts": {
     "fetch:style:spec": "./scripts/download-style-spec.sh",
-    "generate": "yarn node ./scripts/autogenerate",
-    "test": "npm run lint && npm run unittest",
+    "generate": "npx node ./scripts/autogenerate",
```

Why do we need `npx` here?
```suggestion
    "generate": "node ./scripts/autogenerate",
```